Complete the requirements section of the report

apeinot opened the pull request:
Describe the requirement of the function that will be impacted by the refactoring.
We try to stick to the standard for describing the functional requirements.
This should resolve #3

apeinot committed:
Complete the requirements section of the report

Describe the requirement of the function that will be impacted by the refactoring.
We try to stick to the standard for describing the functionnal requirements.
This is linked to #3

Samuelhertzberg reviewed (changes requested):
"| Requirement Description         | The method *identified* edges with output tag and fetches their Next Intra Task Operator Info " => identifies (?)

There also some inconsistency about capital letters. Sometimes there are several in a sentence such as "Very High" and some single word fields are not capitalizedd such as "prepare" and "contructor"

emilelm96 reviewed (changes requested):
I think you should proof-read this once more, since there are some typos and grammatical errors.

apeinot committed:
Fix some typos in requirements description

This should solve #3

Jonagoo committed:
Fixed grammatical errors in requirements

Fixed some grammatical errors in the requirements section of the report. This should resolve #3
